Source/GameActor.cpp

```diff
@@ -70,7 +71,7 @@ void GameActorImage::render(GameActorInstance* gameActorInstance, Renderer2D* re
 	}
 
 	renderer->prep(texture, WhiteColor, renderOpacity(), worldTransform(), m_VertexBuffer, boneInfluenceMatrices(), boneInfluenceMatricesLength(), m_DeformVertexBuffer, seqUVBuffer, uvOffset);
-	renderer->draw(indexBuffer, triangleCount()*3);
+	renderer->draw(indexBuffer, triangleCount()*3, m_IndexOffset);
```

Comment:
This is where I added the index offset back in, and accordingly updated the draw signature and logic.